cmd/cli/cmd/faucet_test.go

```diff
@@ -25,14 +24,18 @@ func testCase5_1(t *testing.T, tc *testCmd) {
 		beenFauceted = append(beenFauceted, bal == "1000000000")
 	}
 
+	var results []string
 	for i := range liteAccounts {
-		commandLine := fmt.Sprintf("faucet %s", liteAccounts[i])
-		_, err := tc.execute(t, commandLine)
+		commandLine := fmt.Sprintf("-j faucet %s", liteAccounts[i])
```

Comment:
Execute adds the -j option already, so redundant here even though benign.

Reply:
Fixed